pyvista/core/dataset.py

```diff
@@ -3,7 +3,6 @@
 from __future__ import annotations
 
 from collections.abc import Iterable
-from collections.abc import Iterator
```

Comment:
Side note: this is an unused import that was introduced in #6835 and not linted (removed) by `ruff` automatically